src/onnx/onnx.cpp

```diff
@@ -437,11 +447,7 @@ struct onnx_parser
             s.visit([&](auto v) { copy(v, std::back_inserter(op.dims)); });
         }
 
-        if(!args[0]->get_shape().standard())
-        {
-            args[0] = prog.add_instruction(op::contiguous{}, args[0]);
-        }
-
+        args[0] = make_contiguous(args[0]);
         return prog.add_instruction(op, args[0]);
```

Comment:
Can we just do `return prog.add_instruction(op, make_contiguous(args[0]));`?

Reply:
changed accordingly